bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -16,6 +22,16 @@
 
 public final class LaunchUtils {
 
+    private static final ILogger logger = Logger.getLogger(LaunchUtils.class);
+
+    private static final ServiceTracker<LaunchListener,LaunchListener> launchListeners;
+
+    static {
+        final BundleContext context = FrameworkUtil.getBundle(LaunchUtils.class).getBundleContext();
```

Comment:
I've reviewed this more and can see that only other classes from same bundle call this.  This context should never be null, at least in its current use I can't see how this class could be loaded without the bundle being started.